### ISSUE TITLE
Feature Request - Table Row's Border Customization

### ISSUE THREAD

**Issue by maksudur-rahman-maruf** [HC200ok/vue3-easy-data-table#48]
Is there any way that We can Increase or Decrease the table row's  border length?   Also Can We give some space between two rows?

**Comment by HC200ok** [comment]
> Is there any way that We can Increase or Decrease the table row's border length

Do you mean  border width？

> Also Can We give some space between two rows?

Do you mean the padding or the row height?

**Comment by maksudur-rahman-maruf** [comment]
how do I create a gap between rows in a table? like this... 
[===ROW===]
[===ROW===]
[===ROW===]
[===ROW===]

**Comment by HC200ok** [comment]
@maksudur-rahman-maruf

> gap between rows

Can you provide a capture with annotation or something else to make it more visually?

**Comment by maksudur-rahman-maruf** [comment]
![unviewed documents](https://user-images.githubusercontent.com/43263023/176151126-42ecf541-b9d3-4c54-a07d-72081f12e308.png)
**I am trying to achieve this...but I have encountered few issues like**
1. First I had to hide the footer. thats okay but I am not able to hide the footer border.
2. I am not able to hide the header border.
3. I am not able to create a gap between rows in a table

**This is my current table looks like**
![right now my table](https://user-images.githubusercontent.com/43263023/176153378-7792897d-d4f1-4f2f-b279-407251191e59.png)
